src/createXcmTypes/ParaToPara.spec.ts

```diff
@@ -0,0 +1,219 @@
+// Copyright 2023 Parity Technologies (UK) Ltd.
+
+import { Registry } from '../registry';
+import { mockParachainApi } from '../testHelpers/mockParachainApi';
+import { ParaToPara } from './ParaToPara';
+
+describe('ParaToPara', () => {
+	const registry = new Registry('kusama', {});
+	describe('Beneficiary', () => {
+		it('Should work for V2', () => {
+			const beneficiary = ParaToPara.createBeneficiary(
+				mockParachainApi,
+				'0xf5d5714c084c112843aca74f8c498da06cc5a2d63153b825189baa51043b1f0b',
+				2
+			);
+
+			const expectedRes = {
+				v2: {
+					parents: 0,
+					interior: mockParachainApi.registry.createType('InteriorMultiLocation', {
```

Comment:
Do we need InteriorMultiLocation here? Can it just be the object. When I did the refactor a while ago I remember that InteriorMultiLocation also resolves only to V2 types, so it could pose issues in the future, especially for V3.